documentation/modules/auxiliary/scanner/ip/ipidseq.md

```diff
@@ -0,0 +1,94 @@
+## Vulnerable Application
+
```

Comment:
not reading more in to this, I think it implies it needs to be done on a local network.  If that is the case, it should be mentioned here.